### ISSUE TITLE
pytorch awesome

### ISSUE THREAD

**Issue by yzbx** [ISCAS007/torchseg#10]
# pytorch
- https://github.com/bharathgs/Awesome-pytorch-list
- https://github.com/ritchieng/the-incredible-pytorch
- https://github.com/ncullen93/torchsample
- https://github.com/meetshah1995/pytorch-semseg
- https://github.com/ZijunDeng/pytorch-semantic-segmentation

**Comment by yzbx** [comment]
# psp_convert -dropout -relu -batchnorm
```
----------------------------------------------------------------
        Layer (type)               Output Shape         Param #
================================================================
            Conv2d-1         [-1, 64, 357, 357]           1,728
            Conv2d-4         [-1, 64, 357, 357]          36,864
            Conv2d-7        [-1, 128, 357, 357]          73,728
        MaxPool2d-10        [-1, 128, 179, 179]               0
           Conv2d-11         [-1, 64, 179, 179]           8,192
           Conv2d-14         [-1, 64, 179, 179]          36,864
           Conv2d-17        [-1, 256, 179, 179]          16,384
           Conv2d-19        [-1, 256, 179, 179]          32,768
       Bottleneck-22        [-1, 256, 179, 179]               0
           Conv2d-23         [-1, 64, 179, 179]          16,384
           Conv2d-26         [-1, 64, 179, 179]          36,864
           Conv2d-29        [-1, 256, 179, 179]          16,384
       Bottleneck-32        [-1, 256, 179, 179]               0
           Conv2d-33         [-1, 64, 179, 179]          16,384
           Conv2d-36         [-1, 64, 179, 179]          36,864
           Conv2d-39        [-1, 256, 179, 179]          16,384
       Bottleneck-42        [-1, 256, 179, 179]               0
           Conv2d-43        [-1, 128, 179, 179]          32,768
           Conv2d-46          [-1, 128, 90, 90]         147,456
           Conv2d-49          [-1, 512, 90, 90]          65,536
           Conv2d-51          [-1, 512, 90, 90]         131,072
       Bottleneck-54          [-1, 512, 90, 90]               0
           Conv2d-55          [-1, 128, 90, 90]          65,536
           Conv2d-58          [-1, 128, 90, 90]         147,456
           Conv2d-61          [-1, 512, 90, 90]          65,536
       Bottleneck-64          [-1, 512, 90, 90]               0
           Conv2d-65          [-1, 128, 90, 90]          65,536
           Conv2d-68          [-1, 128, 90, 90]         147,456
           Conv2d-71          [-1, 512, 90, 90]          65,536
       Bottleneck-74          [-1, 512, 90, 90]               0
           Conv2d-75          [-1, 128, 90, 90]          65,536
           Conv2d-78          [-1, 128, 90, 90]         147,456
           Conv2d-81          [-1, 512, 90, 90]          65,536
       Bottleneck-84          [-1, 512, 90, 90]               0
           Conv2d-85          [-1, 256, 90, 90]         131,072
           Conv2d-88          [-1, 256, 90, 90]         589,824
           Conv2d-91         [-1, 1024, 90, 90]         262,144
           Conv2d-93         [-1, 1024, 90, 90]         524,288
       Bottleneck-96         [-1, 1024, 90, 90]               0
           Conv2d-97          [-1, 256, 90, 90]         262,144
          Conv2d-100          [-1, 256, 90, 90]         589,824
          Conv2d-103         [-1, 1024, 90, 90]         262,144
      Bottleneck-106         [-1, 1024, 90, 90]               0
          Conv2d-107          [-1, 256, 90, 90]         262,144
          Conv2d-110          [-1, 256, 90, 90]         589,824
          Conv2d-113         [-1, 1024, 90, 90]         262,144
      Bottleneck-116         [-1, 1024, 90, 90]               0
          Conv2d-117          [-1, 256, 90, 90]         262,144
          Conv2d-120          [-1, 256, 90, 90]         589,824
          Conv2d-123         [-1, 1024, 90, 90]         262,144
      Bottleneck-126         [-1, 1024, 90, 90]               0
          Conv2d-127          [-1, 256, 90, 90]         262,144
          Conv2d-130          [-1, 256, 90, 90]         589,824
          Conv2d-133         [-1, 1024, 90, 90]         262,144
      Bottleneck-136         [-1, 1024, 90, 90]               0
          Conv2d-137          [-1, 256, 90, 90]         262,144
          Conv2d-140          [-1, 256, 90, 90]         589,824
          Conv2d-143         [-1, 1024, 90, 90]         262,144
      Bottleneck-146         [-1, 1024, 90, 90]               0
          Conv2d-147          [-1, 256, 90, 90]         262,144
          Conv2d-150          [-1, 256, 90, 90]         589,824
          Conv2d-153         [-1, 1024, 90, 90]         262,144
      Bottleneck-156         [-1, 1024, 90, 90]               0
          Conv2d-157          [-1, 256, 90, 90]         262,144
          Conv2d-160          [-1, 256, 90, 90]         589,824
          Conv2d-163         [-1, 1024, 90, 90]         262,144
      Bottleneck-166         [-1, 1024, 90, 90]               0
          Conv2d-167          [-1, 256, 90, 90]         262,144
          Conv2d-170          [-1, 256, 90, 90]         589,824
          Conv2d-173         [-1, 1024, 90, 90]         262,144
      Bottleneck-176         [-1, 1024, 90, 90]               0
          Conv2d-177          [-1, 256, 90, 90]         262,144
          Conv2d-180          [-1, 256, 90, 90]         589,824
          Conv2d-183         [-1, 1024, 90, 90]         262,144
      Bottleneck-186         [-1, 1024, 90, 90]               0
          Conv2d-187          [-1, 256, 90, 90]         262,144
          Conv2d-190          [-1, 256, 90, 90]         589,824
          Conv2d-193         [-1, 1024, 90, 90]         262,144
      Bottleneck-196         [-1, 1024, 90, 90]               0
          Conv2d-197          [-1, 256, 90, 90]         262,144
          Conv2d-200          [-1, 256, 90, 90]         589,824
          Conv2d-203         [-1, 1024, 90, 90]         262,144
      Bottleneck-206         [-1, 1024, 90, 90]               0
          Conv2d-207          [-1, 256, 90, 90]         262,144
          Conv2d-210          [-1, 256, 90, 90]         589,824
          Conv2d-213         [-1, 1024, 90, 90]         262,144
      Bottleneck-216         [-1, 1024, 90, 90]               0
          Conv2d-217          [-1, 256, 90, 90]         262,144
          Conv2d-220          [-1, 256, 90, 90]         589,824
          Conv2d-223         [-1, 1024, 90, 90]         262,144
      Bottleneck-226         [-1, 1024, 90, 90]               0
          Conv2d-227          [-1, 256, 90, 90]         262,144
          Conv2d-230          [-1, 256, 90, 90]         589,824
          Conv2d-233         [-1, 1024, 90, 90]         262,144
      Bottleneck-236         [-1, 1024, 90, 90]               0
          Conv2d-237          [-1, 256, 90, 90]         262,144
          Conv2d-240          [-1, 256, 90, 90]         589,824
          Conv2d-243         [-1, 1024, 90, 90]         262,144
      Bottleneck-246         [-1, 1024, 90, 90]               0
          Conv2d-247          [-1, 256, 90, 90]         262,144
          Conv2d-250          [-1, 256, 90, 90]         589,824
          Conv2d-253         [-1, 1024, 90, 90]         262,144
      Bottleneck-256         [-1, 1024, 90, 90]               0
          Conv2d-257          [-1, 256, 90, 90]         262,144
          Conv2d-260          [-1, 256, 90, 90]         589,824
          Conv2d-263         [-1, 1024, 90, 90]         262,144
      Bottleneck-266         [-1, 1024, 90, 90]               0
          Conv2d-267          [-1, 256, 90, 90]         262,144
          Conv2d-270          [-1, 256, 90, 90]         589,824
          Conv2d-273         [-1, 1024, 90, 90]         262,144
      Bottleneck-276         [-1, 1024, 90, 90]               0
          Conv2d-277          [-1, 256, 90, 90]         262,144
          Conv2d-280          [-1, 256, 90, 90]         589,824
          Conv2d-283         [-1, 1024, 90, 90]         262,144
      Bottleneck-286         [-1, 1024, 90, 90]               0
          Conv2d-287          [-1, 256, 90, 90]         262,144
          Conv2d-290          [-1, 256, 90, 90]         589,824
          Conv2d-293         [-1, 1024, 90, 90]         262,144
      Bottleneck-296         [-1, 1024, 90, 90]               0
          Conv2d-297          [-1, 256, 90, 90]         262,144
          Conv2d-300          [-1, 256, 90, 90]         589,824
          Conv2d-303         [-1, 1024, 90, 90]         262,144
      Bottleneck-306         [-1, 1024, 90, 90]               0
          Conv2d-307          [-1, 256, 90, 90]         262,144
          Conv2d-310          [-1, 256, 90, 90]         589,824
          Conv2d-313         [-1, 1024, 90, 90]         262,144
      Bottleneck-316         [-1, 1024, 90, 90]               0
          Conv2d-317          [-1, 256, 90, 90]       2,359,296
    conv_bn_relu-320          [-1, 256, 90, 90]               0
          Conv2d-322           [-1, 19, 90, 90]           4,883
          Conv2d-323          [-1, 512, 90, 90]         524,288
          Conv2d-326          [-1, 512, 90, 90]       2,359,296
          Conv2d-329         [-1, 2048, 90, 90]       1,048,576
          Conv2d-331         [-1, 2048, 90, 90]       2,097,152
      Bottleneck-334         [-1, 2048, 90, 90]               0
          Conv2d-335          [-1, 512, 90, 90]       1,048,576
          Conv2d-338          [-1, 512, 90, 90]       2,359,296
          Conv2d-341         [-1, 2048, 90, 90]       1,048,576
      Bottleneck-344         [-1, 2048, 90, 90]               0
          Conv2d-345          [-1, 512, 90, 90]       1,048,576
          Conv2d-348          [-1, 512, 90, 90]       2,359,296
          Conv2d-351         [-1, 2048, 90, 90]       1,048,576
      Bottleneck-354         [-1, 2048, 90, 90]               0
       AvgPool2d-355           [-1, 2048, 6, 6]               0
          Conv2d-356            [-1, 512, 6, 6]       1,048,576
        Upsample-359          [-1, 512, 90, 90]               0
       AvgPool2d-360           [-1, 2048, 3, 3]               0
          Conv2d-361            [-1, 512, 3, 3]       1,048,576
        Upsample-364          [-1, 512, 90, 90]               0
       AvgPool2d-365           [-1, 2048, 2, 2]               0
          Conv2d-366            [-1, 512, 2, 2]       1,048,576
        Upsample-369          [-1, 512, 90, 90]               0
       AvgPool2d-370           [-1, 2048, 1, 1]               0
          Conv2d-371            [-1, 512, 1, 1]       1,048,576
        Upsample-374          [-1, 512, 90, 90]               0
transform_psp_caffe-375         [-1, 4096, 90, 90]               0
          Conv2d-376          [-1, 512, 90, 90]      18,874,368
    conv_bn_relu-379          [-1, 512, 90, 90]               0
          Conv2d-381           [-1, 19, 90, 90]           9,747
        Upsample-382         [-1, 19, 713, 713]               0
================================================================
Total params: 68,072,166
Trainable params: 68,072,166
Non-trainable params: 0
----------------------------------------------------------------
```
